flink-ai-flow/lib/notification_service/bin/start_notification_service.py

```diff
@@ -28,22 +28,23 @@
 def start_notification_service(port: int = 50052,
                                db_conn: str = None,
                                enable_ha: bool = False,
-                               server_uris: str = None,
+                               server_uri: str = None,
                                create_table_if_not_exists: bool = True):
     if db_conn:
         storage = DbEventStorage(db_conn, create_table_if_not_exists)
     else:
         raise Exception('Failed to start notification service without database connection info.')
 
     if enable_ha:
-        if not server_uris:
-            raise Exception("When HA enabled, server_uris must be set.")
+        if not server_uri:
+            raise Exception("When HA enabled, server-uri must be set.")
```

Comment:
```suggestion
            raise Exception("When HA enabled, ‘server-uri’ must be multiple addresses, separated by commas.")
```

Reply:
When HA enabled, users are still allowed to pass only one server-uri. As long as this server-uri is accessible, the other HA servers can be discovered.

Reply:
Deleted